app/services/update_attorney_case_review.rb

```
@@ -0,0 +1,93 @@
+class UpdateAttorneyCaseReview
+  include ActiveModel::Model
+
+  validates :id, presence: true
+  validate :authorized_to_edit
+  validate :correct_format
+
+  def initialize(id:, document_id:, user_id:)
+    @id = id
+    @document_id = document_id
+    @user_id = user_id
+  end
+
+  def call
+    success = valid?
+
+    case_review.update!(document_id: document_id) if success
+
+    FormResponse.new(success: success, errors: errors.messages)
+  end
+
+  private
+
+  attr_reader :id, :document_id, :user_id
+  attr_writer :user_id
+
+  def case_review
+    @case_review ||= AttorneyCaseReview.find_by(id: id)
```

Should we throw an error if not found? I usually use `find` method that will throw an exception if the item is not found.